services/daq-blueapi/values.yaml

```diff
@@ -18,11 +18,9 @@ blueapi:
       operator: Equal
       value: bl45p
       effect: NoSchedule
-  service:
-    type: LoadBalancer # TODO: when DNS entry created expose only as ClusterIP
   ingress:
```

Comment:
I assume blue-api chart 0.4.3 already supports 'ingress' value?

Reply:
Yes, it's already supported but was disabled previously only because the Ingress DNS hadn't been made